markdown/tuto-020-shell-tools.md

```diff
@@ -245,16 +252,26 @@ You can run a command on all selected nodes with
 
 this time of course, you cannot specify another set of nodes than the selection.
 
+If you want to directly access a Bash prompt **inside a Docker container** please choose the username `container`, like this :
+
+    ssh container@fit25
+
 ### Saving images
 
 You have the ablility to save the image - this now of course applies only to **one node** at a time. To save node 25
 
-    rsave 25 -o my-imge-name
+    rsave 25 -o my-image-name
 
 This ends up in the common repository `/var/lib/rhubarbe-images`, under a name that holds the hostname and saving time. You can also provide an extra name to `rsave` with the `-o` option.
 
 Images that may be of common interest usually needto be renamed; get in touch with the admins if you need to do this.
 
+### Saving Docker images
+
+If you want to save a Docker container that you may have modified from its base image, you can use the [baleine save](https://github.com/haysberg/baleine/wiki/Save-a-custom-container) command.
+
+    baleine save --node 1 --name mycustomimage:1.0 
+
```

Comment:
would it not make more sense to add this at the bottom of the next tab on baleine ?

Reply:
I moved it now.